backend/api_http.py

```diff
@@ -32,7 +32,7 @@ class EmailSubsStep(str, Enum):
 # These classes serve for the purpose of request body type chechking for FastAPI
 class SingleSubscription(BaseModel):
     visa_type: str
-    code: str
+    code: Union[str, List[str]]
     till: Optional[datetime]
```

Comment:
I haven't found any support yet but I'm concerned about this approach. I usually adopt the practice that a HTTP POST request's body should be deterministic. Can we change the body structure to

```py
class SingleSubscription(BaseModel):
    visa_type: str
    code: List[str]
    till: Optional[datetime]
```

And construct the request body in the frontend accordingly?